endstate_rew/system.py

```diff
@@ -77,7 +80,11 @@ def _seed_velocities(masses: np.array) -> np.ndarray:
 
 
 def initialize_simulation(
-    molecule: Molecule, at_endstate: str = "", platform: str = "CPU"
+    molecule: Molecule,
+    at_endstate: str = "",
+    platform: str = "CPU",
+    w_dir="",
+    conf_id: int = 0,
```

Comment:
yes! I think in one of the next PRs the CHARMM simulation function should also have a keyword that let's us select which conformation to use.
I am saving the system object and indicate from where to reload with the `w_dir` keyword --- this is something that we don't have to do for CHARMM, since I am only doing this because the generation of the partial charges (which only happens for the open-ff systems) takes very long.